README.md

```diff
@@ -1,63 +1,124 @@
-> "Well, in our country," said Alice, still panting a little, "you'd generally 
-> get to somewhere else—if you run very fast for a long time, as we've been 
+> "Well, in our country," said Alice, still panting a little, "you'd generally
+> get to somewhere else—if you run very fast for a long time, as we've been
 > doing."
 >
-> "A slow sort of country!" said the Queen. "Now, here, you see, it takes all 
-> the running you can do, to keep in the same place. If you want to get 
+> "A slow sort of country!" said the Queen. "Now, here, you see, it takes all
+> the running you can do, to keep in the same place. If you want to get
 > somewhere else, you must run at least twice as fast as that!"
 >
 > [Carroll, Lewis: Through the Looking-Glass, Chapter 2](
     https://www.gutenberg.org/files/12/12-h/12-h.htm)
 
-## About
+<br>
+
+<h1>About</h1>
+
 The Red Queen benchmark framework was created to facilitate the benchmarking
 algorithms used in quantum compilation.
 
-The framework is tightly integrated into `pytest`.  Therefore, to use it 
-effectively, you should know the basics of `pytest` first. Take a look at the 
+The framework is tightly integrated into `pytest`.  Therefore, to use it
+effectively, you should know the basics of `pytest` first. Take a look at the
 [introductory material](https://docs.pytest.org/en/latest/getting-started.html).
 
-## Usage
+
+<br>
+
+<h1>Usage</h1>
+
+
 Red Queen is a framework for benchmarking quantum compilation algorithms. Since
-it was not designed as a package, there is no notion of installation. Hence, you
-must clone this repository to use it:
+is is still in early development, you must clone this repository to use it:
+
+
 ```bash
 git clone git@github.com:Qiskit/red-queen.git
 ```
 
-To run benchmarks, you must first go to the `red-queen` directory and install 
+To run benchmarks, you must first go to the `red-queen` directory and install
 the required packages:
+
+
 ```bash
 cd red-queen
-pip install -r requirements.txt
+pip install red-queen
```

Comment:
This isn't quite correct. It should be:

```suggestion
pip install .
```

`pip install red-queen` will try to download red-queen from pypi (and fail because it's not published there) as it's not a directory after you're in the repository.